workflow/tests/base_results/real_homes.csv

```diff
@@ -1,82 +1,82 @@
-house073.xml,32.27,115.74,26.83,11.78,95.03999999999999,15.14,26.8,23.13,13.969999999999999,32.44,0.0,47.55,0.16596,65.0,1.0,65.0,54.0,54.0,1,29.81,132.38,26.36,11.78,108.27,15.469999999999999,26.83,23.13,13.969999999999999,32.44,0.0,47.55,0.16596
-house074.xml,73.53,13.58,33.1,7.53,10.03,19.49,8.67,6.39,13.6,6.53,0.0,22.22,0.2416,60.0,1.0,60.0,74.0,74.0,0,71.96,14.7,33.21,7.53,10.76,19.92,8.67,6.39,13.6,6.53,0.0,22.22,0.2416
-house075.xml,65.41,30.91,15.83,11.39,49.43,10.01,19.41,31.37,5.5,18.5,0.0,17.85,0.21679,60.0,1.0,60.0,65.0,65.0,0,62.19,34.82,15.78,11.39,55.62,10.25,19.43,31.37,5.5,18.5,0.0,17.85,0.21679
-house076.xml,62.82,7.56,5.09,5.79,12.05,3.65,12.37,9.21,2.09,7.23,0.0,10.84,0.19729,64.0,1.0,64.0,63.0,63.0,1,64.27,6.8,5.18,5.79,10.85,3.66,12.36,9.21,2.09,7.23,0.0,10.84,0.19729
-house077.xml,71.94,11.56,5.09,8.66,18.5,3.68,9.54,9.25,1.99,7.33,0.0,17.87,0.22468,64.0,1.0,64.0,72.0,72.0,0,75.33,9.52,5.36,8.66,15.23,3.76,9.53,9.25,1.99,7.33,0.0,17.87,0.22468
-house078.xml,58.95,28.72,55.59,8.99,46.0,32.15,16.34,23.53,17.94,9.48,0.0,28.36,0.26439,65.0,1.0,65.0,59.0,59.0,1,56.23,32.77,56.12,8.99,52.48,33.53,16.34,23.53,17.94,9.48,0.0,28.36,0.26439
-house079.xml,48.07,44.55,2.52,10.35,31.09,1.28,12.36,11.84,0.11,3.23,0.0,22.67,0.21761,69.0,1.0,69.0,48.0,48.0,1,44.28,53.14,2.15,10.35,36.71,1.1099999999999999,12.36,11.84,0.11,3.23,0.0,22.67,0.21761
-house080.xml,42.48,31.22,62.34,8.99,22.68,36.99,10.83,6.3,13.86,1.83,0.0,25.26,0.27847,62.0,1.0,62.0,42.0,42.0,1,40.22,36.31,63.17,8.99,26.06,39.04,10.84,6.3,13.86,1.83,0.0,25.26,0.27847
-house081.xml,58.91,4.86,4.7,5.6,4.32,2.72,6.34,1.83,0.68,5.53,0.0,9.17,0.16377,59.0,1.0,59.0,59.0,59.0,1,58.64,5.0,4.69,5.6,4.42,2.73,6.34,1.83,0.68,5.53,0.0,9.17,0.16377
+XML,ERI,REUL Heating (MBtu),REUL Cooling (MBtu),REUL Hot Water (MBtu),EC_r Heating (MBtu),EC_r Cooling (MBtu),EC_r Hot Water (MBtu),EC_x Heating (MBtu),EC_x Cooling (MBtu),EC_x Hot Water (MBtu),EC_x Dehumid (MBtu),EC_x L&A (MBtu),IAD_Save (%),ES Reference Home ERI,ES SAF (Size Adjustment Factor),ES SAF Adjusted ERI Target,ES Rated Home ERI,ES Rated Home ERI w/o OPP,ES ENERGY STAR Certification,IECC ERI,IECC REUL Heating (MBtu),IECC REUL Cooling (MBtu),IECC REUL Hot Water (MBtu),IECC EC_r Heating (MBtu),IECC EC_r Cooling (MBtu),IECC EC_r Hot Water (MBtu),IECC EC_x Heating (MBtu),IECC EC_x Cooling (MBtu),IECC EC_x Hot Water (MBtu),IECC EC_x Dehumid (MBtu),IECC EC_x L&A (MBtu),IECC IAD_Save (%)
+house001.xml,58.12,11.51,2.11,6.49,18.39,2.24,12.66,9.34,1.32,9.82,0.0,10.84,0.23187,63.0,1.0,63.0,58.0,58.0,1.0,59.96,10.43,2.2,6.49,16.66,2.27,12.65,9.34,1.32,9.82,0.0,10.84,0.23187
+house002.xml,75.06,51.66,21.18,9.54,41.02,12.11,10.56,31.45,6.55,9.45,0.0,24.05,0.18385,65.0,1.0,65.0,75.0,75.0,0.0,69.36,60.02,21.18,9.54,47.79,12.71,10.56,31.45,6.55,9.45,0.0,24.05,0.18385
```

Comment:
It looks like the ZERH results are not making their way into this file. This may have been my bug. You can fix it by adding ZERH to [this](https://github.com/NREL/OpenStudio-ERI/blob/1aab5e5fca8ffad00e12dbf701ffcabe4092b0f3/workflow/tests/real_homes_test.rb#L33-L36). (Compare with [this](https://github.com/NREL/OpenStudio-ERI/blob/1aab5e5fca8ffad00e12dbf701ffcabe4092b0f3/workflow/tests/energy_rating_index_test.rb#L34-L38), which correctly adds ZERH to the CI results.)